applications/DEMApplication/custom_constitutive/DEM_D_Hertz_viscous_Coulomb_CL.cpp

```diff
@@ -121,12 +121,11 @@ namespace Kratos {
 
         const double equiv_mass = 1.0 / (1.0/my_mass + 1.0/other_mass);
 
-        const double my_gamma    = element1->GetProperties()[DAMPING_GAMMA];
-        const double other_gamma = element2->GetProperties()[DAMPING_GAMMA];
-        const double equiv_gamma = 0.5 * (my_gamma + other_gamma);
+        Properties& properties_of_this_contact = element1->GetProperties().GetSubProperties(element2->GetProperties().Id());
+        const double damping_gamma = properties_of_this_contact[DAMPING_GAMMA];
 
-        const double equiv_visco_damp_coeff_normal     = 2.0 * equiv_gamma * sqrt(equiv_mass * mKn);
-        const double equiv_visco_damp_coeff_tangential = 2.0 * equiv_gamma * sqrt(equiv_mass * mKt);
+        const double equiv_visco_damp_coeff_normal     = 2.0 * damping_gamma * sqrt(equiv_mass * mKn);
+        const double equiv_visco_damp_coeff_tangential = 2.0 * damping_gamma * sqrt(equiv_mass * mKt);
```

Comment:
so, stupid question: This might cause current results to be wrong? Even already published ones?

Reply:
and this is not influenced by the new changes?
https://github.com/KratosMultiphysics/Kratos/blob/1285b884b329b355d5c687159091d2ba400f87a6/applications/DEMApplication/custom_constitutive/DEM_D_Hertz_viscous_Coulomb_CL.cpp#L149